src/quick-start.md

```diff
@@ -153,7 +153,8 @@ kind: ClusterAdmissionPolicy
 metadata:
   name: privileged-pods
 spec:
-  module: registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.5
+  policyServer: reserved-instance-for-tenant-a
```

Comment:
I would not change this here. The original structure of the quickstart was:

- Install
- Describe what a PolicyServer resource is
- Describe what a ClusterAdmissionPolicy resource is
- Enforce your first policy (without targeting any specific PolicyServer)

Given this page is going an overhaul in https://github.com/kubewarden/docs/pull/73/files, I think it's fine to fix the broken example, but changing this here requires deploying the PolicyServer described before, whereas we wanted to show this deployment as simple as possible (no need to specify a policyServer, it will be defaulted if missing).

I'll defer to @nunix for further approval/merge, given he's working on this overhaul.

Reply:
It's a bit a caveat here as, in terms of example, running the commands will work (I did it several times the past days).

However I kept the `yaml` examples as is, as they were used "only" to explain the options inside (transformed in a table in the overhaul).
So while the `yaml` examples are there, in the quick start they are not used.

Still, I totally get the versioning problem and in this case, would you prefer to transform it into something generic (again, it's only used for explanations), such as: `module: registry://ghcr.io/kubewarden/policies/pod-privileged:<version tag>`

Hope this makes sense.